hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OmMetadataGenerator.java

```diff
@@ -449,6 +499,26 @@ private void applyOperation(long counter) throws Exception {
           }
       );
       break;
+    case EMPTY_RPC:
+      getMetrics().timer(operation.name()).time(() -> {
+            try {
+              ozoneManagerClient.echoRPCReq(new byte[] {}, 0, false);
+            } catch (IOException e) {
+              throw new RuntimeException(e);
+            }
+          }
+      );
+      break;
+    case EMPTY_RPC_WRITE_RATIS:
```

Comment:
I think we are also doing this as part of `OmRPCLoadGenerator`.

Reply:
Removed